app/views/request/select_authority.html.erb

```diff
@@ -36,7 +37,8 @@ $(document).ready(function() {
     <% if AlaveteliConfiguration.allow_batch_requests && @user && @user.can_make_batch_requests? %>
       <div id="batch_request_link" class="batch_request_link">
         <p>
-          <%= _('Or make a <a href="{{url}}">batch request</a> to <strong>multiple authorities</strong> at once.',
+          <%= _('Or make a <a href="{{url}}">batch request</a> to <' \
+                   'strong>multiple authorities</strong> at once.',
```

Comment:
I'd have put the opening angle bracket on the same line as the rest of the tag here